pom.xml

```diff
@@ -369,6 +369,16 @@
 				<artifactId>hapi-structures-v26</artifactId>
 				<version>2.1</version>
 			</dependency>
+			<dependency>
+				<groupId>dom4j</groupId>
```

Comment:
I removed this dependency since it was unused and most importantly because its vulnerable with no fix according to dependabot. If you reintroduce it please use an up to date one like https://github.com/dom4j/dom4j/releases/tag/version-2.1.3

Reply:
I tried dom4j 2.1.3 but that version is not available in the OpenMRS Nexus repository. Do you know how to get it there pls?

Reply:
Actually that was wrong, dom4j 2.1.3 is available under the (changed) group id `org.dom4j`